lib/route.js

```diff
@@ -99,6 +99,10 @@ function buildRouting (options) {
 
   // Convert shorthand to extended route declaration
   function prepareRoute (method, url, options, handler) {
+    if (url && typeof url !== 'string') {
```

Comment:
When would `url` not be defined?

Reply:
Hmm. Valid Point ! Thanks for noticing. Even if it is undefined by any chance this if condition 's typeof check would handle it. Hence not required. Removing it straightaway !